### DISCUSSION TITLE
Why are we leaving Bonsai presocial exp workflows running 24/7?

### DISCUSSION THREAD

**Discussion by jkbhagatio** [SainsburyWellcomeCentre/aeon_experiments#184] [General]
Asked by @lorycalca:

"I am not sure I fully understand why we are not stopping Bonsai at least at the end of each day. I do understand that we want the workflow to stay on for the day to avoid typing errors from the experimenter but why overnight? Are we collecting 12h+ of video without the mice which are unneccesary? I personally find it a waste but happy to hear the reasoning behind this choice."

**Comment by jkbhagatio** [comment]
I think one continued important test of the exp workflows is to leave them running 24/7 (even though we are only running mice ~9 hours/day at max) as a test for the upcoming 24/7 experiments. 

It is true we are collecting a lot of data that is not useful for behavioural analysis, but this data can be useful for testing / troubleshooting the acuqisition pipelines. It is good to know if we experience some issues in a 24/7 continuous setting that we may not experience in shorter settings, and how often things like harp sync issues, dropped camera frames, etc... occur in this 24/7 continuous setting.

**Reply by glopesdev** [reply]
@jkbhagatio @lorycalca indeed, the reasoning here was to make sure that the camera issues are indeed all resolved, both stability and latency-wise. Once we are confident the new setup is stable we can definitely stop collecting data like this.

Also, if storage is ever an issue, we could in principle write a crawler that deletes truly unused chunks of data.
